app/services/budgetkey-item.ts

```diff
@@ -46,21 +46,34 @@ export class BudgetKeyItemService {
     Promise.reject(new Error('No layout for ' + path));
   }
 
-  getItemData(query: string, headersOrder: string[]): Promise<object> {
-    let url = 'http://next.obudget.org/api/query?query=' +
-      encodeURIComponent(query);
+  private _budgetNumberFormatter(value: any) {
+    return value
+      ? parseFloat(value).toLocaleString('he-IL', { style: 'currency', currency: 'ILS' })
+      : '-';
```

Comment:
this will show '-' for zeros as well, no?

Reply:
also `parseFloat` may return `NaN`. I think, it's better to parse value first (`value = parseFloat(value)`), and then check it: `isFinite(value) ? value.toLocaleString(...) : '-'`